plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/utils/actions.ts

```diff
@@ -117,11 +116,19 @@ export const createKycAccountSession = async (
 	apiURL: string,
 	poEligible: boolean
 ): Promise< AccountKycResult > => {
+	const selfAssessmentData = fromDotNotation( data );
+	const requestData: Record< string, unknown > = {
+		progressive: isPoEligible,
```

Comment:
```suggestion
		progressive: poEligible,
```
otherwise it won't be passed. Tried to push it but some weird pre-commit checks didn't allow me to